test/data/model-properties.js

```diff
@@ -107,5 +108,6 @@ export default {
     setupConfig: {},
     bandwidthEstimate: null,
     bitrateSelection: null,
-    backgroundLoading: Features.backgroundLoading
+    backgroundLoading: Features.backgroundLoading,
+    language: getPlayerLanguage(),
```

Comment:
It's not optional if it's always added by the player. Don't we default it to 'en'?